_posts/2021-06-10-save-the-date-MV-2021.md

```diff
@@ -0,0 +1,47 @@
+---
+title: "Save the date, werde Vorstand, leite die Wahl bei der Mitgliederversammlung 2021"
+layout: post
+author: "Stephan Druskat"
+menulang: de
+---
+
+Die **ordentliche Mitgliederversammlung 2021 des *de-RSE e.V.*** wird stattfinden am
+
+> **Mittwoch, 15. September 2021, von 13:00 bis ca. 16:00 Uhr**.
+
+Frist- und formgerechte Einladungen folgen natürlich noch.
+
+## Offene Vorstandsrollen - bring Dich ein!
+
+Für die Wahl des Vorstands ergibt sich in diesem Jahr mindestens eine **neu zu besetzende Rolle: stellvertretender Schriftführer**.
+Ich werde in dieser Wahl nicht kandidieren.
```

Comment:
Vorschlag: "Stephan Druskat, der dieses Amt derzeit inne hat, wird in der nächsten Wahl nicht kandidieren. Darüber hinaus kannst Du dich natürlich auch für alle anderen Vorstandsämter zur Wahl stellen - der aktuelle Vorstand freut sich über jede Unterstützung und die Chance, sein Engagement im Verein auch ohne Amt fortzusetzen."

Reply:
```suggestion
Stephan Druskat, der dieses Amt derzeit inne hat, wird in der nächsten Wahl nicht kandidieren. Darüber hinaus kannst Du Dich natürlich auch für alle anderen Vorstandsämter zur Wahl stellen - der aktuelle Vorstand freut sich über jede Unterstützung und die Chance, sein Engagement im Verein auch ohne Amt fortzusetzen.
```
Danke, das ist gut!